spark-jobs/src/main/scala/filodb/repair/PartitionKeysCopier.scala

```diff
@@ -54,15 +54,22 @@ class PartitionKeysCopier(conf: SparkConf) {
   private val sourceCassConfig = sourceConfig.getConfig("cassandra")
   private val targetCassConfig = targetConfig.getConfig("cassandra")
   private val sourceDataset = conf.get("spark.filodb.partitionkeys.copier.source.dataset")
+  private val sourceDatasetConfig = datasetConfig(sourceConfig)
+  private val targetDatasetConfig = datasetConfig(targetConfig)
   private val sourceDatasetRef = DatasetRef.fromDotString(sourceDataset)
   private val targetDatasetRef = DatasetRef.fromDotString(conf.get("spark.filodb.partitionkeys.copier.target.dataset"))
   private val sourceSession = FiloSessionProvider.openSession(sourceCassConfig)
   private val targetSession = FiloSessionProvider.openSession(targetCassConfig)
 
-  private val numOfShards: Int = getShardNum
+  val schemas = Schemas.fromConfig(sourceConfig).get
+  private[repair] def partKeyHashFn = (partKey: PartKeyRecord) =>
+    Option(schemas.part.binSchema.partitionHash(partKey, UnsafeUtils.arayOffset))
```

Comment:
this is incorrect, it should be
`Option(schemas.part.binSchema.partitionHash(partKey.partKey, UnsafeUtils.arayOffset))`

Reply:
fixed in 9491bcd

Reply:
fixed in 9491bcd